Dockerfile

```diff
@@ -2,6 +2,11 @@ FROM registry.centos.org/kbsingh/openshift-nginx:latest
 
 MAINTAINER Jean-Francois Chevrette <jfchevrette@gmail.com>
 
+# Labels consumed by the build service
+LABEL Component="nginx-redirector" \
```

Comment:
I think the linter wants Name and Version. Not sure what Component is but maybe it's not mendatory?